### DISCUSSION TITLE
Let's discuss pbench!

### DISCUSSION THREAD

**Discussion by portante** [distributed-system-analysis/pbench#2073] [General]
Hey folks, just starting a discussion about Pbench!

**Comment by dbutenhof** [comment]
Sounds interesting! Should we talk about no-install non-root agent containers, PCP data curation, or what?

Is there anyone listening?

**Comment by ndokos** [comment]
I am lurking...

**Comment by ndokos** [comment]
LC_ALL=C.UTF-8 as done in the server unit tests does not seem to work on RHEL7 - is that correct? Is that too low-level for discussion here?

**Reply by dbutenhof** [reply]
Probably should be a separate discussion thread. No idea how that works, yet, if the concept even exists.

**Reply by portante** [reply]
@ndokos, for something that technical, I'd suggest just filing an issue.
